src/app/@core/models/models.ts

```diff
@@ -83,6 +84,24 @@ export interface IAnalyzerConfig {
   observable_supported?: string[];
 }
 
+export interface IRawConnectorConfig {
+  [name: string]: IConnectorConfig;
+}
+
+export interface IConnectorConfig {
```

Comment:
We should have 2 interfaces. An `IAbstractConfig` interface which could be `extended` into 2: `IConnectorConfig` and `AnalyzerConfig`.